evergreen/install_baas.sh

```diff
@@ -454,7 +461,7 @@ echo "Adding fake appid to skip baas server drop optimization"
 echo "Starting baas app server"
 
 "${WORK_PATH}/baas_server" \
-    --configFile=etc/configs/test_config.json --configFile="${BASE_PATH}/config_overrides.json" > "${BAAS_SERVER_LOG}" 2>&1 &
+    --configFile=etc/configs/test_config.json --configFile=etc/configs/test_rcore_config.json > "${BAAS_SERVER_LOG}" 2>&1 &
```

Comment:
Tyler requested that the BAAS team maintain the config overrides for our tests so that they can keep them up to date with server features. I've removed our local overrides file `evergreen/config_overrides.json` and now this points to a config in their repo https://github.com/10gen/baas/blob/master/etc/configs/test_rcore_config.json

Reply:
Nice! I will eventually move our dependency definitions (e.g. `STITCH_SUPPORT_LIB_URL`) to use the baas definitions as well, but they need to add the definitions for Ubuntu before this happens.